### ISSUE TITLE
dependency to hippo-cms-testutils is not available

### ISSUE THREAD

**Issue by dennisvonk** [bloomreach/docs-feedback#24]
https://documentation.bloomreach.com/14/library/development/suppress-log-messages-during-unit-tests.html

"add a dependency on hippo-cms-testutils" doesn't work. So all the example code (which is not complete either) won't work.

**Comment by nvankampenhout** [comment]
Thanks for your feedback, I marked this page for review.

**Comment by nvankampenhout** [comment]
Hi, it will probably only be after the 14.2 release that we'll be able to schedule some time to properly review this documentation page but I got some quick feedback from the engineering team:

The hippo-cms-testutils package is available in the [public Maven repository](https://maven.onehippo.com/maven2/org/onehippo/cms7/hippo-cms-testutils/) and included in the dependency management in the community release POM. So if your project uses the release POM, Maven should be able to download it.

If you have any more details that could help us understand why it is not working for you that would be much appreciated.

**Comment by dennisvonk** [comment]
Hi Niels,

I tried lots of things before I sent the mail. And it just wouldn't work.
But when I removed my local repository I noticed more things weren't
working. So it turned out my maven settings weren't correct anymore.
Changed my settings back lead to a working download of that library (and
others).

Thanks for coming back to me. Sorry about this obvious mistake on my part.

Dennis Vonk
Consultant Digital Experience

*t: *+31202219819
*m: *+31621561667
*w: *www.incentro.com

Olympisch Stadion 3  | 1076 DE  |  Amsterdam



Op di 5 mei 2020 om 16:18 schreef Niels van Kampenhout <
notifications@github.com>:
